xmppserver/src/main/java/org/jivesoftware/openfire/OfflineMessageStore.java

```diff
@@ -168,6 +177,7 @@ public void addMessage(Message message) {
             size += msgXML.length();
```

Comment:
Just another trivial comment, can the update to sizeCache too for this user be avoided, in case of a db write failure?